packages/twenty-front/src/modules/workflow/components/RightDrawerWorkflowEditStep.tsx

```diff
@@ -0,0 +1,10 @@
+import { showPageWorkflowSelectedNodeState } from '@/workflow/states/showPageWorkflowSelectedNodeState';
+import { useRecoilValue } from 'recoil';
+
+export const RightDrawerWorkflowEditStep = () => {
+  const showPageWorkflowSelectedNode = useRecoilValue(
+    showPageWorkflowSelectedNodeState,
+  );
+
+  return <p>{showPageWorkflowSelectedNode}</p>;
```

Comment:
do you need <p> ? I would prefer <> and some design if needed

Reply:
This component is a placeholder for now. It was used to prove that selecting a node worked. It's styled in my next PR and the `<p>` tag will disappear.

I understood that `<p>` tags should be prohibited.